File: WeChatAuto4_X/WebSocketServer/Server/WebSockets/Models/MessagePackage.cs
using System.Net.WebSockets;
using System.Text.Json;
using System.Text.Json.Serialization;

/// <summary>
/// 微信消息包
/// </summary>
public class MessagePackage
{
    [JsonPropertyName("request_id")]
    public string? RequestId { get; set; }
    [JsonPropertyName("func_Name")]
    public string? FuncName { get; set; }
    [JsonPropertyName("options")]
    public string? Options { get; set; }
    [JsonPropertyName("from")]
    public required string From {get;set;}
}
/// <summary>
/// 微信消息 - 包装类
/// </summary>
public class MessagePackageWrapper : MessagePackage
{
    public WebSocketHandler? handler { get; set; }

    public static MessagePackageWrapper Create(MessagePackage package, WebSocketHandler handler)
    {
        return new MessagePackageWrapper
        {
            handler = handler,
            RequestId = package.RequestId,
            FuncName = package.FuncName,
            Options = package.Options,
            From = package.From,
        };
    }
}

File: WeChatAuto4_X/wechatauto_py/src/wechat_auto_sdk/models/message_package.py
from pydantic import BaseModel

class MessagePackage(BaseModel):
    # 请求id,每次请求唯一一个id
    request_id: str
    # 内部调用函数
    func_Name: str
    # 函数参数
    options: str
    # 通过哪个微信来发送,目的最主要是为了支持多微信
    from_wechat: str